modules/programs/anki.nix

```diff
@@ -0,0 +1,567 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}:
+let
+  ankiBaseDir =
+    if pkgs.stdenv.hostPlatform.isDarwin then
+      "Library/Application Support/Anki2"
+    else
+      "${config.xdg.dataHome}/Anki2";
+  cfg = config.programs.anki;
+  # This script generates the Anki SQLite settings DB using the Anki Python API.
+  # The configuration options in the SQLite database take the form of Python
+  # Pickle data.
+  # A simple "gldriver6" file is also generated for the `videoDriver` option.
+  buildAnkiConfig = pkgs.writers.writeText "buildAnkiConfig" ''
```

Comment:
Might feel less bloated if we move some this module to a directory structure and put these into a separate helper/lib file.

- anki/
  - default.nix
  - lib/helper.nix